src/uu/echo/src/echo.rs

```diff
@@ -117,6 +117,8 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
 
     let matches = App::new(executable!())
         .name(NAME)
+        .setting(clap::AppSettings::TrailingVarArg)
```

Comment:
Could you please add a comment explaining why we need this? thanks

Reply:
sure